prime-router/src/test/csv_test_files/expected/simplereport-az.csv

```diff
@@ -1,51 +1,51 @@
 Sending_Application,Sending_Application_ID,Lab_Name,Lab_CLIA,Lab_Address,Lab_City,Lab_State,Lab_Zip,Lab_Phone,Patient_ID,Patient_First_Name,Patient_Last_Name,Patient_Date_of_Birth,Patient_Sex,Race,Ethnicity,Patient_Street_Address,Patient_City,Patient_State,Patient_Zip,Patient_County,Patient_Phone_Number,Ordering_facility_name,Ordering_Facility_Address,Ordering_Facility_City,Ordering_Facility_State,Ordering_Facility_Zip,Provider_First_Name,Provider_Last_Name,Provider_Phone_Number,Specimen_ID,Collection_Date,Specimen_Type,Specimen_Site,Test_Code,Test_Name,Result,Result_Date,Notes
```

Comment:
Because AZ was our first state, we use AZ schemas for our test data comparing actuals to expecteds.  So when we change the AZ schema, we have to change the expected test data too.